src/Cli/dotnet/commands/dotnet-workload/WorkloadCommandBase.cs

```diff
@@ -116,7 +116,8 @@ protected bool VerifySignatures
                 nugetLogger,
                 Reporter,
                 restoreActionConfig: RestoreActionConfiguration,
-                verifySignatures: VerifySignatures);
+                verifySignatures: VerifySignatures,
+                isNuGetTool: true);
```

Comment:
Thanks for the minimal change! I think what we should do here is instead rename this parameter to something like "`shouldApplyPackageSourceMapping`" instead, because the only usage of `_isNuGetTool` in this file is to trigger PSM. As it is, workload commands shouldn't need to think about .NET SDK Tools at all.